go.mod

```diff
@@ -284,8 +284,8 @@ replace (
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
 	github.com/gorilla/rpc => github.com/dymensionxyz/rpc v1.3.1
 	github.com/libp2p/go-libp2p-pubsub => github.com/dymensionxyz/go-libp2p-pubsub v0.0.0-20231220125656-9e904c648367
+	github.com/osmosis-labs/osmosis/v15 => github.com/dymensionxyz/osmosis/v15 v15.2.0-dymension-v1.1.3
```

Comment:
why u need it?

Reply:
I keep getting errors locally saying that osmosis 15.2.1 (from dymension) doesn't exist.
I guess it would be better to just change the version in Dymension